query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/ServerGuesser.java

```diff
@@ -63,15 +63,15 @@
 
     @SuppressWarnings("StaticInitializerReferencesSubClass")
     private static final Map<String, Function<Connection, ServerGuesser>> GUESSER_LOOKUP =
-            ImmutableMap.of(LdapTypeField.ACTIVE_DIRECTORY,
+            ImmutableMap.of(LdapTypeField.ActiveDirectory.ACTIVE_DIRECTORY,
                     ServerGuesser.ADGuesser::new,
-                    LdapTypeField.EMBEDDED,
+                    LdapTypeField.EmbeddedLdap.EMBEDDED,
```

Comment:
Everywhere you use these static finals from the enums looks like you've leaked knowledge of their internals to me. I'm not sure there's a better way around this, but it's not an appealing design.